### DISCUSSION TITLE
Is there a possibility to remove fonts from Collabora?

### DISCUSSION THREAD

**Discussion by g-work** [CollaboraOnline/online#10213] [Q&A] [answered]
My useres complain that the font list is too crowded. Indeed it's very long becaus of all these Noto Sans versions. Is there any possibility to remove fonts? I already checked `/opt/lool/systemplate/usr/share/fonts/truetype`, `/usr/share/fonts/truetype` etc. but can't find them.

**Comment by vmiklos** [comment]
Feel free to ask at https://forum.collaboraonline.com/ if you don't get an answer here.

**Comment by g-work** [comment, marked as answer]
Currently there is no option to remove fonts but you can use a custom directory for fonts and only add the ones you like to be available. Haven't tried so far.

See https://forum.collaboraonline.com/t/removing-limiting-available-fonts/2982/3
